pkg/prowgen/prowgen_tests.go

```diff
@@ -166,7 +166,7 @@ func DiscoverTests(r Repository, openShift OpenShift, sourceImageName string, sk
 			preSubmitConfiguration.RunIfChanged = test.RunIfChanged
 			cfg.Tests = append(cfg.Tests, *preSubmitConfiguration)
 
-			if !test.SkipCron && !openShift.CandidateRelease {
+			if !test.SkipCron && !openShift.SkipCron && !openShift.CandidateRelease {
```

Comment:
this now is a bit confusing, double "skipCron". Can we somehow align this?

Reply:
The `test.SkipCron` is for a single test only. The `openShift.SkipCron` if for all tests running on the given OpenShift version. You might require running periodic job for a given OpenShift version but only skip generating the cron job for a single test.

Would a comment in code make it more clear?

Reply:
Yeah, I think that explanation was missing for me. If you can explain this in a comment, would be great.

Reply:
Added clarification, please let me know if this is clearer.